libcontainer/seccomp/seccomp_linux.go

```diff
@@ -86,6 +86,15 @@ func InitSeccomp(config *configs.Seccomp) (int, error) {
 		}
 	}
 
+	// Enable libseccomp binary tree optimization.
+	if len(config.Syscalls) > 8 {
```

Comment:
Do we need a comment "why 8?" or does git blame suffice?

Reply:
Yeah there should be a comment here IMHO.

Reply:
added